src/UlozenkaLib/APIv3/Formatter/JsonFormatter.php

```diff
@@ -410,6 +443,49 @@ private function proccessStatusHistoryResponseData($dataObject)
     /**
      *
      * @param stdClass $dataObject
+     *
+     * @return \UlozenkaLib\APIv3\Model\Tracking
+     */
+    private function proccessTrackingResponseData($dataObject)
+    {
+        if (isset($dataObject[0])) {
+            $c = $dataObject[0];
+            $consignmentJsonAttr = $this->getJsonAttr($c, 'consignment');
+
+            $consignment = new \UlozenkaLib\APIv3\Model\Tracking\Consignment(
+                $this->proccessLinks($this->getJsonAttr($consignmentJsonAttr, '_links')),
+                $this->getJsonAttr($consignmentJsonAttr, 'id'),
+                $this->getJsonAttr($consignmentJsonAttr, 'partner_consignment_id'),
+                $this->getJsonAttr($consignmentJsonAttr, 'cash_on_delivery'),
+                $this->getJsonAttr($consignmentJsonAttr, 'currency'),
+                $this->getJsonAttr($consignmentJsonAttr, 'parcel_count'),
+                $this->getJsonAttr($consignmentJsonAttr, 'card_payment_allowed')
+            );
+
+            $statusesJsonAttr = $this->getJsonAttr($c, 'statuses');
+            $statuses = [];
+            foreach ($statusesJsonAttr as $statusJsonAttr) {
+                $datetimeJsonAttr = $this->getJsonAttr($statusJsonAttr, 'date');
+                $datetime = $this->proccessDateTime($datetimeJsonAttr);
+                $status = new \UlozenkaLib\APIv3\Model\Tracking\Status(
+                    $this->proccessLinks($this->getJsonAttr($statusJsonAttr, '_links')),
+                    $this->getJsonAttr($statusJsonAttr, 'id'),
+                    $this->getJsonAttr($statusJsonAttr, 'name'),
+                    $this->getJsonAttr($statusJsonAttr, 'tracking_name'),
+                    $datetime
+                );
+                $statuses[] = $status;
+            }
+
+            return new Tracking($consignment, $statuses);
+        }
+
+        return new Tracking(NULL);
```

Comment:
Druhý parametr je not null, ale pokud se aplikuje moje poznámka výše, bude se tu stejně jednat o prázdné pole. Hodnotu null v rámci Uloženky zapisujeme podle PSR malými písmeny, stejně tak boolean hodnoty true/false.